transport/server.cc

```diff
@@ -207,11 +207,68 @@ cql_server::cql_server(distributed<cql3::query_processor>& qp, auth::service& au
         sm::make_derive("requests_shed", _stats.requests_shed,
                         sm::description("Holds an incrementing counter with the requests that were shed due to overload (threshold configured via max_concurrent_requests_per_shard). "
                                             "The first derivative of this value shows how often we shed requests due to overload in the \"CQL transport\" component.")),
-       sm::make_gauge("requests_memory_available", [this] { return _memory_available.current(); },
+        sm::make_gauge("requests_memory_available", [this] { return _memory_available.current(); },
                         sm::description(
                             seastar::format("Holds the amount of available memory for admitting new requests (max is {}B)."
                                             "Zero value indicates that our bottleneck is memory and more specifically - the memory quota allocated for the \"CQL transport\" component.", _max_request_size))),
+        
+        sm::make_derive("server_errors", _stats.errors[exceptions::exception_code::SERVER_ERROR],
+                        sm::description("Counts the total number of raised CQL SERVER errors.")),
```

Comment:
nitpick: I don't know if the word "raised" is the right term. It's errors "returned" to the client, no? Although I guess "raised" will be understood as well, so maybe it's ok.

Reply:
Not very adamant on the choice of that word. In fact, you might be right that 'returned' is more straightforward in its meaning